test/connect.js

```diff
@@ -29,27 +29,27 @@ suite("Credentials", function() {
   }
 
   test("no creds", function(done) {
-    var parts = urlparse('amqp://localhost');
+    var parts = new URL('amqp://localhost');
     var creds = credentialsFromUrl(parts);
     checkCreds(creds, 'guest', 'guest', done);
   });
   test("usual user:pass", function(done) {
```

Comment:
can we add test case for the vhost, using the 
`amqp://${config.username}:${config.password}@${config.hostname}:${config.port}/${config.vhost}` structure?